stable/dask-distributed/values.yaml

```diff
@@ -0,0 +1,42 @@
+# Default values for dask.
+# This is a YAML-formatted file.
+# Declare name/value pairs to be passed into your templates.
+# name: value
+
+Scheduler:
+  Name: scheduler
+  Image: "daskdev/dask"
+  ImageTag: "latest"
+  Replicas: 1
+  Component: "dask-scheduler"
+  Cpu: "500m"
+  Memory: "1024Mi"
+  ServicePort: 8786
+  ContainerPort: 8786
+
+WebUI:
+  Name: webui
+  ServicePort: 8787
+  ContainerPort: 8787
+
+Worker:
+  Name: worker
+  Image: "daskdev/dask"
+  ImageTag: "latest"
+  Replicas: 3
+  Component: "dask-worker"
+  Cpu: "500m"
+  Memory: "1024Mi"
```

Comment:
Yes, mili-cpu basically half CPU for each container. What defaults do you think are better?